doc/foresight.rst

```diff
@@ -8,21 +8,47 @@
 Foresight Options
 #####################
 
+Planning horizons and warm starts
```

Comment:
I would eliminate use of term "warm-start" due to ambiguity with solver warmstart.
```suggestion
Planning horizons
```